stupoli.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+volumes:
+  local_postgres_data: {}
+  local_postgres_data_backups: {}
+
+services:
+  django:
+    build:
+      context: .
+      dockerfile: ./compose/local/django/Dockerfile
+    image: osler_local_django
+    container_name: django
+    depends_on:
+      - postgres
+    volumes:
+      - .:/app
+    env_file:
+      - ./.envs/.local/.django
+      - ./.envs/.local/.postgres
+      - ./.envs/.local/.stupoli
+    ports:
+      - "8000:8000"
+    command: /start
+    links:
+      - selenium
+
+  postgres:
+    build:
+      context: .
+      dockerfile: ./compose/production/postgres/Dockerfile
+    image: osler_production_postgres
+    container_name: postgres
+    volumes:
+      - local_postgres_data:/var/lib/postgresql/data
+      - local_postgres_data_backups:/backups
+    env_file:
+      - ./.envs/.local/.postgres
+
+  selenium:
+    image: selenium/standalone-chrome-debug:3.141
+    ports:
+      - 4444:4444   # Selenium
+      - 5900:5900   # VNC server
```

Comment:
There's no reason for you to know this, but the only docker compose yml files that are used are `local.yml` and `production-umkc.yml`. We should make a `production-stupoli.yml` file that is based on `production-umkc.yml` but that includes a VM for nginx. For historical reasons the UMKC deployment's nginx is running on the host machine but this is suboptimal imo. @wwick do you agree?

Reply:
It was my idea to create production-umkc and local-umkc. I actually use local-umkc.yml a ton in order to test the UMKC settings fo hiding certain elements of the UI locally.

Reply:
I think when I was working on this issue with @ericsallinger , there's no good way to inherit .yml files, and you need to specify the settings in the .yml file. Since we modify .yml so infrequently, we don't have to repeat ourselves very often, although I agree that it's not ideal.

Reply:
> local-umkc.yml a ton in order to test the UMKC settings

Great! This is very smart--actually one of the big advantages of docker is supposed to be that your local and production environments can be almost identical. We should put some thought into how to make this a bit more scalable as we expand to more institutions.